### ISSUE TITLE
standard_init_linux.go:187: exec user process caused "exec format error"

### ISSUE THREAD

**Issue by Magicloud** [rancher/os#2463]
**RancherOS Version: (ros os version)** v1.4.0

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)** RaspBerryPi 3

```
[rancher@rancher ~]$ sudo ros config syslinux
standard_init_linux.go:187: exec user process caused "exec format error"
```

**Comment by niusmallnan** [comment]
The RPi image use this bootloader https://github.com/DieterReuter/rpi-bootloader.
So this command cannot work on RPi.

We should give some friendly tips.

**Comment by niusmallnan** [comment]
The RPi image use this bootloader https://github.com/DieterReuter/rpi-bootloader.
So this command cannot work on RPi.

We should give some friendly tips.

**Comment by kingsd041** [comment]
Fixed in rancheros v1.4.2-rc1
We have provided some friendly tips:
```
[root@rancher ~]# ros -v
version v1.4.2-rc1 from os image rancher/os:v1.4.2-rc1
[root@rancher ~]# sudo ros config syslinux
raspberry pi can not use this command
```
If it is [Booting from ISO](https://rancher.com/docs/os/v1.x/en/installation/running-rancheros/workstation/boot-from-iso/) or [Using Docker Machine](https://rancher.com/docs/os/v1.x/en/installation/running-rancheros/workstation/docker-machine/), it will prompt: `global.cfg can not be found`

**Comment by niusmallnan** [comment]
backported to v1.4.2
